apps/api/v2/src/modules/webhooks/inputs/create-webhook.input.ts

```diff
@@ -0,0 +1,37 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { WebhookTriggerEvents } from "@prisma/client";
+import { IsArray, IsBoolean, IsEnum, IsOptional, IsString } from "class-validator";
+
+export class CreateWebhookInputDto {
+  @IsString()
+  @IsOptional()
+  payloadTemplate?: string;
+
+  @IsBoolean()
+  @IsOptional()
+  active?: boolean = true;
+
+  @IsString()
+  subscriberUrl!: string;
+
+  @IsArray()
+  @ApiProperty({
+    example: [
+      "BOOKING_CREATED",
+      "BOOKING_RESCHEDULED",
+      "BOOKING_CANCELLED",
+      "BOOKING_CONFIRMED",
```

Comment:
do we want to keep them in caps lock, instead of having `bookingCreated` etc using camelCase or `booking-created`

Reply:
yes I think we should keep it as is because that is how they have been sent in the webhook payload to customers for a long time